src/Symfony/Component/Ldap/README.md

```diff
@@ -0,0 +1,23 @@
+Ldap Component
+=============
+
+A Ldap client for PHP on top of PHP's ldap extension.
+
+This component also provides a stub for the missing
+`ldap_bind` function in PHP versions lower than 5.6.
```

Comment:
Where is this stub? And on the php documentation it says ldap_bind is avaialble sine php 4?